{
    "error": "An error occurred: HTTPSConnectionPool(host='mangatak.com', port=443): Max retries exceeded with url: /manga/6935974387-disaster-class-necromancer (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001AD95348920>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))"
}